client/src/app.tsx

```diff
@@ -93,7 +93,11 @@ function LoadingFallback({ message }: { message?: string }) {
 export function App(appProps) {
   useDebugGA();
 
-  const homePage = CRUD_MODE ? <WritersHomepage /> : <Homepage {...appProps} />;
+  // When preparing a build for use in the NPM package, CRUD_MODE is always true.
+  // But if the App is loaded from the code that builds the SPAs, then `isServer`
+  // is true. So you have to have `isServer && CRUD_MODE` at the same time.
+  const homePage =
+    !isServer && CRUD_MODE ? <WritersHomepage /> : <Homepage {...appProps} />;
```

Comment:
The rule is simple. *Any* component that is of the style...
```
const MyComponent = React.lazy(...)
```
...can never be used when `isServer` is true, which it is during the `yarn prepare-build` because that includes the command `yarn build --spas-only` which runs SSR on all SPAs.